build/gen_stub.php

```diff
@@ -2715,13 +2858,12 @@ function initPhpParser() {
 $context->forceRegeneration = isset($options["f"]) || isset($options["force-regeneration"]);
 $context->forceParse = $context->forceRegeneration || $printParameterStats || $verify || $generateClassSynopses || $replaceClassSynopses || $generateMethodSynopses || $replaceMethodSynopses;
 
-$targetClassSynopses = $argv[$optind + 1] ?? null;
-if ($replaceClassSynopses && $targetClassSynopses === null) {
+$targetSynopses = $argv[$argc - 1] ?? null;
```

Comment:
Since multiple directories can be passed in to gen_stubs, the class/method synopsis target should be the last one of them.